config/config_test.go

```diff
@@ -67,6 +67,46 @@ func TestBlacklist(t *testing.T) {
 	}
 }
 
+func TestArgsBlacklist(t *testing.T) {
+	customArgsBlacklist := []string{
+		"consul_token",
+		"dd_password",
+		"blocked_from_yaml",
+	}
+
+	defaultRegexs := CompileStringsToRegex(defaultArgsBlacklist)
+	customRegexs := CompileStringsToRegex(customArgsBlacklist)
+	mergedRegexs := append(defaultRegexs, customRegexs...)
+	t.Log("default regexp", defaultRegexs)
+	t.Log("custom regexp", customRegexs)
+	t.Log("merged regexp", mergedRegexs)
+
+	cases := []struct {
+		cmdline       []string
+		parsedCmdline []string
+	}{
+		{[]string{"agent", "-password", "1234"}, []string{"agent", "-password", "********"}},
```

Comment:
How does this play for casing of the arguments?  e.g. `[]string{"agent", "-PASSWORD", "1234"}`

Might be worth adding a test case or two for that as well. If it's not handled, should it be?